src/main/java/repositories/FacultyRepository.java

```diff
@@ -6,6 +6,6 @@
 import java.util.Collection;
 
 public interface FacultyRepository extends JpaRepository<Faculty,Long> {
-    Collection<Faculty> findByNameContains(String part);
+    Collection<Faculty> findByNameOrColourIgnoreCase(String part);
```

Comment:
У этого метода должно быть 2 параметра:
```suggestion
    Collection<Faculty> findByNameOrColourIgnoreCase(String name, String colour);
```